pkg/api/store/nodetemplate/store.go

```diff
@@ -24,3 +31,39 @@ func (s *Store) Delete(apiContext *types.APIContext, schema *types.Schema, id st
 	}
 	return s.Store.Delete(apiContext, schema, id)
 }
+
+func (s *Store) Create(apiContext *types.APIContext, schema *types.Schema, data map[string]interface{}) (map[string]interface{}, error) {
+	driver := configfield.GetDriver(data)
+	if driver == "" {
+		id := convert.ToString(data["credentialId"])
+		if id == "" {
+			return nil, httperror.NewAPIError(httperror.MissingRequired, "a Config field or credentialId must be set")
```

Comment:
is `Config` the exact field name here? if not, we should somehow better reflect it